omap-tutorial/README.md

```diff
@@ -139,6 +139,9 @@ Go to omap-tutorial folder.
 export LD_LIBRARY_PATH=/usr/local/lib
 
 test-platform ./PandaBoardES.xml
+ # Disable criteria check at XML load time in order to enable test without criteria.
+remote-process localhost 5001 setFailureOnFailedSettingsLoad false
```

Comment:
It is more for people who wants to start test platform without criteria. At list here you manage to lunch it with the new setting file.